File: 7. TopGamesApp/app/src/main/java/com/example/topgames/MainActivity.java
package com.example.topgames;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    ArrayList<TopGames> gamesList;

    RecyclerView recyclerView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        gamesList = new ArrayList<>();
        gamesList.add(new TopGames("Horizon Chase",R.drawable.card1));
        gamesList.add(new TopGames("PUBG",R.drawable.card2));
        gamesList.add(new TopGames("Head Ball 2",R.drawable.card3));

        gamesList.add(new TopGames("Fifa 2022",R.drawable.card5));
        gamesList.add(new TopGames("Fortnite",R.drawable.card6));
        gamesList.add(new TopGames("Hooked on You",R.drawable.card4));

        recyclerView = findViewById(R.id.recyclerView);


        TopGamesAdapter topGamesAdapter = new TopGamesAdapter(gamesList,this);

        RecyclerView.LayoutManager layoutManager = new LinearLayoutManager(this,LinearLayoutManager.VERTICAL,false);

        recyclerView.setLayoutManager(layoutManager);
        recyclerView.setAdapter(topGamesAdapter);



    }
}

File: __Kotlin Apps/Vaccines App/app/src/main/java/com/example/vaccinesapp/MainActivity.kt
package com.example.vaccinesapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val rv : RecyclerView = findViewById(R.id.RV)
        rv.layoutManager = LinearLayoutManager(this,LinearLayoutManager.VERTICAL,false)


        //creating data
        val vaccineList : ArrayList<VaccineModel> = ArrayList()
        val v1 = VaccineModel("Vikas",R.drawable.ic_launcher_background)
        val v2 = VaccineModel("is",R.drawable.ic_launcher_background)
        val v3 = VaccineModel("here",R.drawable.ic_launcher_background)
        val v4 = VaccineModel("r",R.drawable.ic_launcher_background)
        val v5 = VaccineModel("u",R.drawable.ic_launcher_background)
        vaccineList.add(v1)
        vaccineList.add(v2)
        vaccineList.add(v3)
        vaccineList.add(v4)
        vaccineList.add(v5)

        //initializing Adapter
        val myAdapter:VaccineAdapter = VaccineAdapter(vaccineList)
        rv.adapter = myAdapter



    }
}